docs/sources/flow/monitoring/debugging.md

```diff
@@ -16,8 +16,18 @@ Follow these steps to debug issues with Grafana Agent Flow:
 Grafana Agent Flow includes an embedded UI viewable from Grafana Agent's HTTP
 server, which defaults to listening at `http://localhost:12345`.
 
-> The documentation for the [`grafana-agent run`][grafana-agent run] command describes how to
-> modify the address Grafana Agent listens on for HTTP traffic.
+> **NOTE**: For security reasons, installations of Grafana Agent Flow on
+> non-containerized platforms default to listening on `localhost`. default
+> prevents other machines on the network from being able to view the UI.
+>
+> To expose the UI to other machines on the network on non-containerized
+> platforms, refer to the documentation for how you [installed][install]
```

Comment:
[installed][install] link doesn't render

Reply:
Do you think it would help if I defined the link?

Reply:
😃  <- windows smile specifically

happy friday!